services/net/reporting/ReportingManager.cs

```diff
@@ -703,7 +703,7 @@ private async Task GenerateReportAsync(ReportRequestModel request, API.Areas.Ser
         await HandleChesEmailOverrideAsync(request.RequestorId);
 
         var contexts = new List<EmailContextModel>();
-        if (!String.IsNullOrWhiteSpace(request.To))
+        if (!String.IsNullOrWhiteSpace(request.To) && request.To.IsValidEmail())
```

Comment:
The downside to this is it will ignore invalid email addresses and will hide a possible issue.